charts/apisix/templates/pdb.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if (and .Values.apisix.enabled .Values.apisix.podDisruptionBudget.enabled) }}
+apiVersion: policy/v1beta1
```

Comment:
Should we also consider the `policy/v1`?

Reply:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

`v1` available since v1.21, I think it's better to migrate to `v1` later this/next year (2022/2023)

Reply:
OK, gotcha.

By the way, please do not resolve conversation by yourself, it should be closed by the reviewer, or we cannot find the place we have doubts easily when we review this PR in the next time, thanks!